examples/child_window.rs

```diff
@@ -77,7 +77,7 @@ fn main() {
     })
 }
 
-#[cfg(not(all(os_linuxy, x11)))]
+#[cfg(not(all(free_unix, x11_platform)))]
```

Comment:
x11_platform implies free_unix, so you can just use it. The same for wayland. So you'd have just wayland/x11 in the code, and not free unix. Use free_unix only when we're really aiming for that and not for x11/wayland.